### PR TITLE
*: prepare to enforce errcheck, part1

### PR DESCRIPTION
**Pull request by coocood**
See #4668

### CODE REVIEW

**Comment by coocood** [comment]
@winkyao @zz-jason PTAL

File: cmd/benchkv/main.go [hunk 1]
```diff
@@ -68,16 +69,17 @@ func Init() {
 	driver := tikv.Driver{}
 	var err error
 	store, err = driver.Open(fmt.Sprintf("tikv://%s?cluster=1", *pdAddr))
-	if err != nil {
-		log.Fatal(err)
-	}
+	terror.Log(err)
```
Comment (shenli) [inline cmd/benchkv/main.go]: This should be MustNil().

**Comment by zz-jason** [comment]
LGTM

**Comment by shenli** [comment]
LGTM

**Pull request merged**